docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -4,6 +4,8 @@
 <titleabbrev>Infrastructure threshold</titleabbrev>
 ++++
 
+//TODO: Change this to inventory rule.
```

Comment:
I will rename this rule later because I'm not sure GitHub will give us a sensible diff after the rename.